zilencer/management/commands/populate_db.py

```diff
@@ -177,8 +179,22 @@ def handle(self, **options):
                                  Recipient.objects.filter(type=Recipient.STREAM)]
 
         # Extract a list of all users
-        user_profiles = [user_profile.id for user_profile in UserProfile.objects.all()] # type: List[int]
-
+        user_profiles = [user_profile for user_profile in UserProfile.objects.all()] # type: List[Any]
+
+        # Populate users with some bar data
+        for user in user_profiles:
+            month = random.choice(range(1, 13)) # type: Any
+            day = random.choice(range(1, 29)) # type: Any
+            random_status = 2 # type: int
+            random_date = datetime.datetime(2015, month, day) # type: datetime.datetime
+            random_client = random.choice(range(1, 3)) # type: Any
+            if random_client == 1:
+                random_client = get_client("API")
+            else:
+                random_client = get_client("website")
+            UserPresence.objects.get_or_create(user_profile=user, client=random_client, timestamp=random_date, status=random_status)
+
+        user_profiles = [user_profile.id for user_profile in UserProfile.objects.all()]
```

Comment:
No need to hit the DB again, just use `user_profile_ids = [user_profile.id for user_profile in user_profiles]`.